docs/installation/helm.md

```diff
@@ -1,75 +1,157 @@
-```
-If you want to install using <strong>Minikube</strong>, follow the steps below:
-#### Prerequisites
-Before we start the installation, make sure you have the following tools installed on your system:
-- [Kubernetes command-line interface (CLI)](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Kubernetes command-line interface (kubectl)](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 - [Helm CLI](https://helm.sh/docs/helm/)
 - [Minikube](https://minikube.sigs.k8s.io/)
-- Docker
+- [Docker](https://docs.docker.com/get-docker/)
 
-#### Step 1: Installing Minikube
-Minikube is a command-line tool that helps you set up a single-node Kubernetes cluster within a virtual machine on your local system. You can install Minikube using a package manager or by following the official installation guide. 
+## Step-by-Step Installation Guide
 
+### Step 1: Installing Minikube
 
-#### Step 2: Installing kubectl and Helm
+Minikube is a tool that helps you set up a single-node Kubernetes cluster within a virtual machine on your local system.
 
-Before we proceed, let's ensure you have both KubeCTL and HelmCLI installed. 
+- Follow the instructions [here](https://minikube.sigs.k8s.io/docs/start/) to install Minikube using a package manager or the official installation guide.
 
-Open your terminal and check the versions of Docker and Helm with the following commands:
+### Step 2: Installing Docker, kubectl and Helm
 
-```
-$ docker version
-```
-Example Output:
-```
-Client:
- Cloud integration: v1.0.31
- Version: 20.10.24
-```
+Before proceeding, ensure you have both kubectl and Helm CLI installed. Open your terminal and check the versions of Docker, kubectl, and Helm:
 
+To install Docker, kubectl, and Helm on different operating systems, you can use the following commands:
+
+Certainly! Here are the commands to install Docker, kubectl, and Helm on Windows, MacOS, and Linux:
+
+<details>
+<summary>Windows</summary>
+
+#### Docker:
+Download and install Docker Desktop from the official Docker website: [Docker Desktop for Windows](https://www.docker.com/products/docker-desktop)
+
+#### kubectl:
+You can install kubectl on Windows using the following PowerShell command:
+
+```powershell
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.22.2/bin/windows/amd64/kubectl.exe
+Move-Item -Path .\kubectl.exe -Destination C:\Windows\System32\kubectl.exe
 ```
-$ helm version
-```
-Example Output:
+
+#### Helm:
+- Installing Helm on Windows
+- Go to the Helm releases page at https://github.com/helm/helm/releases.
+- Download the latest Windows executable file (ending with ".exe").
+- Double-click the downloaded file to start the installation wizard.
+- Follow the prompts to complete the installation.
+
+</details>
+
+<details>
+<summary>MacOS</summary>
+
+#### Docker:
+   
+Download and install Docker Desktop from the official Docker website: [Docker Desktop for Mac](https://www.docker.com/products/docker-desktop)
```

Comment:
Avoid this. Install docker on mac using brew. There is some weird bug with the .dmg installer where stops any running containers abrubutly. Use the below command.

```
brew install --cask docker
```

Reply:
Made the changes, but It turns out this is a MacOS powered by Intel chipset problem afaik.